.github/workflows/sync-version-with-upstream.yml

```
@@ -22,7 +22,15 @@ jobs:
         with:
           token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
           update-script: |
+            #!/bin/bash
             # Update the snapcraft.yaml to match upstream version
             API="https://api.github.com/repos/jenkinsci/jenkins/releases/latest"
-            VERSION=$(curl -s "${API}" | grep -i name | awk 'FNR == 3 {print $2}' | cut -d ',' -f1)
-            sed -i 's/^\(version: \).*$/\1'"$VERSION"'/' snap/snapcraft.yaml
+            VERSION=$(curl -s "${API}" | grep -i name | awk 'FNR == 3 {print $2}' | cut -d ',' -f1 | sed 's/"//g')
+            CRAFT=$(grep -i version ~/snapcraft.yaml | awk '{print $2}' | sed 's/"//g')
```

```suggestion
            CRAFT=$(cat snap//snapcraft.yaml | jq -r '.version')
```

The path here doesn't seem right, and I think `jq` will make this more readable.

---

This has disappeared due to the force push, but I still think the `jq` version is worth considering for usability :)

---

Maybe another time? I just tested the method you're proposing, and it'll require some more rework to get the right results.